contrib/dockerd-rootless.sh

```diff
@@ -77,6 +80,8 @@ if [ -z $_DOCKERD_ROOTLESS_CHILD ]; then
 	# * /run: copy-up is required so that we can create /run/docker (hardcoded for plugins) in our namespace
 	exec $rootlesskit \
 		--net=$net --mtu=$mtu \
+		--slirp4netns-sandbox=$DOCKERD_ROOTLESS_ROOTLESSKIT_SLIRP4NETNS_SANDBOX \
+		--slirp4netns-seccomp=$DOCKERD_ROOTLESS_ROOTLESSKIT_SLIRP4NETNS_SECCOMP \
```

Comment:
What is the default if no `auto` is set? If we omit these flags, would it not use sandbox or seccomp?


Basically wondering if we need the `DOCKERD_ROOTLESS_ROOTLESSKIT_SLIRP4NETNS_SANDBOX ` env-vars or if we should omit these flags in the script (assuming people install the latest version of rootless kit)

Reply:
The rootlesskit default value is currently `false` but planned to be `auto` in future https://github.com/rootless-containers/rootlesskit/blob/v0.7.0/cmd/rootlesskit/main.go#L63-L72

Reply:
Ah, thanks for explaining; yes, `auto` sounds like a better default